utility.py

```diff
@@ -75,7 +76,7 @@ def blueprint_opportunity_formatter(
     """Helper function which serves as a data extraction blueprint for specific formatting"""
 
     div = content.find_all("div", class_=div_elem)
-    command_line_value = extract_command_value()
+    command_line_value = extract_command_value().days_needed
```

Comment:
Use a more specific variable name for the command line value. Such as `days_needed`.

Reply:
Resolved in <https://github.com/acmcsufoss/acmOpportunities/pull/19/commits/7751f596eb159fd801dcf7991a5e09e4e30c69c0>.